### PR TITLE
Website: First incarnation. Very barebones.

### PR DESCRIPTION
**Pull request by SamirTalwar**
Screen Shot

I'll want to get a short screencast on here once we redesign a little.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <title>Codename: Snoopy</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Codename:<br />Snoopy</h1>
+
+      <p>Immediate feedback on your React components, as you develop.</p>
+
+      <p class="code"><code>npx @prodo-ai/snoopy</code></p>
```
Comment (andrejak) [inline docs/index.html]: I think someone already mentioned yesterday that we're not sure if we should be recommending npx due to slowness. It is more user friendly though so I don't have strong feelings either way.

File: docs/style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 2rem;
+  background: #282c34;
```
Comment (andrejak) [inline docs/style.css]: Are we planning to link these styles to the existing theme at all? Or give them their own theme/structure?

File: docs/index.html (hunk 1 above)
Comment (SamirTalwar) [reply docs/index.html]: I think @coffee-cup is right: if people run through `npx`, do a sneaky `npm install --save-dev`/`yarn add --dev` for them.

File: docs/style.css (hunk 2 above)
Comment (SamirTalwar) [reply docs/style.css]: Nah. GitHub Pages won't compile our docs. They need to be just straight-up client-side code.

**Pull request merged**